### ISSUE TITLE
Instruction Tuning hyperparameters for SEED-LLaMA

### ISSUE THREAD

**Issue by dhdbsrlw** [AILab-CVC/SEED#39]
Hello, I am attempting to perform MultiModal Instruction Tuning on the pretrained-8b SEED-LLaMA model. However, I have not found any detailed hyperparameters in your paper or the GitHub README. **Could you share a summary of the instruction tuning hyperparameters for SEED-LLaMA** similar to those provided for the pretraining step?
